package.json

```diff
@@ -28,11 +28,12 @@
     "docs:lint": "prettier '**/*.md' --check",
     "docs:lint:fix": "prettier '**/*.md' --write",
     "test": "lerna run test --concurrency 1",
-    "test:unit": "vitest --project unit --project unit-mainnet",
-    "test:browsers": "vitest --project browser",
-    "test:e2e": "vitest --project e2e",
+    "test:constants": "vitest run --project constants-minimal --project constants-mainnet",
+    "test:unit": "vitest run --project unit-minimal --project unit-mainnet",
+    "test:browsers": "vitest run --project browser",
+    "test:e2e": "vitest run --project e2e-minimal --project e2e-mainnet",
     "download-spec-tests": "lerna run download-spec-tests",
-    "test:spec": "vitest --project spec",
+    "test:spec": "vitest run --project spec-minimal --project spec-mainnet",
```

Comment:
I noticed with the `run` the runner is just stuck and needs to be forced exited, curious if this was omitted by design or is just an oversight cc @nazarhussain

Reply:
@nazarhussain wondering here if we wanna run all spec tests (mainnet and minimal) or just minimal here? I never use top-level command to run these locally

and in CI we do have separate jobs anyways

https://github.com/ChainSafe/lodestar/blob/2ff17840058bb0d4e0a1a9af739fb86d4faf3d7c/.github/workflows/test.yml#L219-L224

Reply:
> Better to keep run explicit here.

I just noticed it locally that it won't exit and even CTRL+C does not work, so you need to kill the shell, so yes seems like vitest changes behavior in CI